includes/class-integrate-convertkit-wpforms.php

```diff
@@ -421,6 +419,46 @@ public function process_entry( $fields, $entry, $form_data, $entry_id = 0 ) { //
 
 	}
 
+	/**
+	 * Returns the subscriber state to use when creating a subscriber.
+	 *
+	 * @since   1.7.3
+	 *
+	 * @param   string $resource_type  Resource Type (subscriber,form,tag,sequence).
+	 * @return  string                  Subscriber state
+	 */
+	private function get_subscriber_state( $resource_type ) {
```

Comment:
I would change to something like `get_initial_subscriber_state` so there's no confusion that this is the state you want to save the subscriber with.